src/pkg_exporter/textfile.py

```diff
@@ -50,12 +59,91 @@ def main():
     rebootmanager.query()
     reboot_gauge.set(rebootmanager.getMetricValue())
 
-    exporter_file = os.getenv("PKG_EXPORTER_FILE", "/var/prometheus/pkg-exporter.prom")
+
+def write_registry_to_file(registry, exporter_file=None):
+    if not exporter_file:
```

Comment:
I think this is duplicate, exporter_file is already filled with a default in line 87